emukit/core/loop/outer_loop.py

```diff
@@ -103,15 +103,21 @@ def _update_models(self):
         for model_updater in self.model_updaters:
             model_updater.update(self.loop_state)
 
-    def get_next_points(self, results: List[UserFunctionResult]) -> np.ndarray:
+    def get_next_points(self,
+        results: List[UserFunctionResult],
+        context: dict={}
+        ) -> np.ndarray:
```

Comment:
That's not a whole lot of params, so I'd keep it in a single line

Reply:
done